### ISSUE TITLE
CloudStorageAccount.Parse throws exception on "UseDevelopmentStorage=true;" connection string under net45 

### ISSUE THREAD

**Issue by dlmelendez** [Azure/azure-storage-net#294]
The Development Storage environment configuration is not available for .NET Framework 4.5. Seems like it should be since some legacy web libraries like Microsoft.Identity.Core 2.2.1 require net45.
Many entries in the CloudStorageAccount class are marked !PORTABLE excluding the development storage from being able to load from the net45 build.
https://raw.githubusercontent.com/Azure/azure-storage-net/v7.0.2/Lib/Common/CloudStorageAccount.cs
```
#if !PORTABLE
            // devstore case
            if (MatchesSpecification(
                settings,
                AllRequired(UseDevelopmentStorageSetting),
                Optional(DevelopmentStorageProxyUriSetting)))
            {
                string proxyUri = null;
                if (settings.TryGetValue(DevelopmentStorageProxyUriSettingString, out proxyUri))
                {
                    accountInformation = GetDevelopmentStorageAccount(new Uri(proxyUri));
                }
                else
                {
                    accountInformation = DevelopmentStorageAccount;
                }

                accountInformation.Settings = ValidCredentials(settings);
                return true;
            }
#endif
```

**Comment by esummers-msft** [comment]
Why are you targeting PORTABLE? net45 should work fine with the Development Storage environment configuration if you target the desktop dll instead of the PCL.

**Comment by dlmelendez** [comment]
Looks like there was a problem adding the reference to the project when the nuget package was added to the project. Somehow, the portable path was pulled in instead of the desktop reference. Then when I debugged into it, it pulled me into the wrong code path. I tried it again on a clean project and the correct project reference path was added for net45 to the desktop dll. Sorry to bother you guys. This one is on me. Closing the issue. Thanks!
